test/integration/filters/udp_listener_echo_filter.cc

```diff
@@ -25,11 +25,11 @@ UdpListenerEchoFilter::UdpListenerEchoFilter(Network::UdpReadFilterCallbacks& ca
     : Network::UdpListenerReadFilter(callbacks) {}
 
 void UdpListenerEchoFilter::onData(Network::UdpRecvData& data) {
-  ENVOY_LOG(trace, "UdpEchoFilter: Got {} bytes from {}", data.buffer_->length(),
-            data.peer_address_->asString());
+  ENVOY_LOG(trace, "UdpEchoFilter: Got {} bytes from {} on interface {}", data.buffer_->length(),
```

Comment:
nit: Remove "interface"?  (local_address is a socket address, not an interface.)